src/Start.php

```diff
@@ -118,6 +119,17 @@ public static function getEndPoint($name)
     return self::$baseURL . self::$endpoints[$name];
   }
 
+  /**
+  * deals with capture URI with charge ID
+  *
+  * @return string endpoint URI for capture
+  */
+
+  public static function getChargeEndPoint($chargeID, $name)
```

Comment:
the idea is to use existing getEndPoint method. but instead of passing $name there we have to pass resource url like "/charges" or "/charges/{id}/capture".

then all underlying methods (curl, etc.) will work automatically